dessia_common/workflow/blocks.py

```diff
@@ -751,27 +751,32 @@ class ModelAttribute(Block):
     :param position: Position of the block in canvas.
     """
 
-    def __init__(self, attribute_name: str, name: str = '', position: Tuple[float, float] = None):
-        self.attribute_name = attribute_name
+    def __init__(self, attribute_type: AttributeType[Type], name: str = '', position: Tuple[float, float] = None):
+        self.attribute_type = attribute_type
```

Comment:
We will need to add a retrocompatibility and warn user that the signature has changed